tasks/main.yml

```diff
@@ -4,20 +4,20 @@
     name: sudo
     update_cache: yes
     state: present
-  when: "{{ ansible_os_family == 'RedHat' }}"
+  when: ansible_os_family == 'RedHat'
 
 - name: Ensure sudo is installed for Debian family OSs
   apt:
     name: sudo
     update_cache: yes
     state: present
-  when: "{{ ansible_os_family == 'Debian' }}"
+  when: ansible_os_family == 'Debian'
 
 - name: Ensure sudo is installed for SUSE family OSs
   zypper:
     name: sudo
     state: present
-  when: "{{ ansible_os_family == 'SUSE' }}"
+  when: ansible_os_family == 'SUSE'
```

Comment:
In order to make this following best practices can you wrap it with template tags?